hw/mcu/dialog/da1469x/syscfg.yml

```diff
@@ -113,6 +113,16 @@ syscfg.defs:
             data before writing, i.e. each such transfer will be split into
             chunks of this size. Buffer is created on stack.
         value: 128
+    QSPI_BURSTCMDA_INIT_VAL:
+        description: >
+            Initialization value for QSPI_BURSTCMDA_REG. The default value
+            is compatible with Macronix flash devices.
```

Comment:
In fact this is compatible with range of devices including Macronix, Winbond, and Gigadevice.
Macronix really wants to have A5 (10100101) while Winbond and Gigadevice want to have xx10xxxx pattern that is included in A5.